index.html

```diff
@@ -5529,16 +5537,20 @@ <h3>Thing Description JSON-LD Context</h3>
     "id": "urn:dev:ops:32473-WoTLamp-1234",
     "title": "MyLampThing",
     "@type": "saref:LightSwitch",
+    "saref:hasState": {
+        "@id": "urn:dev:ops:32473-WoTLamp-1234/state",
+        "@type": "saref:OnOffState"
+    },
     "securityDefinitions": {"basic_sc": {
         "scheme": "basic",
         "in": "header"
     }},
     "security": ["basic_sc"],
     "properties": {
         "status": {
+            "ssn:forProperty": "urn:dev:ops:32473-WoTLamp-1234/state",
```

Comment:
Where is the ssn namespace defined?

Reply:
was a mistake, thanks! Fixed by 957294e.